views/breederDets.handlebars

```diff
@@ -0,0 +1,5 @@
+<section id="posts">
+  {{#each posts}}
+  {{> breederDetails}}
+  {{/each}}
```

Comment:
For the detail partials we should get rid of the `each` and just pass one object instead of an array